pkg/util/fs/proc/proc.go

```diff
@@ -144,7 +144,10 @@ func ReadIDMap(path string) (uint32, uint32, error) {
 	defer r.Close()
 
 	scanner := bufio.NewScanner(r)
-	scanner.Scan()
+	tok := scanner.Scan()
+	if !tok {
```

Comment:
Can be simplified a bit with `if !scanner.Scan()`, since `tok` is used for this single check only